src/classes/RP_Constants.cls-meta.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ApexClass xmlns="http://soap.sforce.com/2006/04/metadata">
+    <apiVersion>40.0</apiVersion>
```

Comment:
We prefer keeping all of our files at the same metadata api version, so we can consciously test and move up all files at the same time.  we've run into issues in the past when some files are newer and others are older.  Currently, we are set on api version 37, so if you could change all of your files to that version, that would be great.  thanks.